setup.cfg

```diff
@@ -48,4 +48,4 @@ python_requires = >=3.8
 
 [options.extras_require]
 all =
-    dwave-gate==0.3.1
+    dwave-gate==0.3.2
```

Comment:
Did we want to promote `dwave-gate` at the same time?

Reply:
We could, should we? Unless anyone thinks otherwise.

Reply:
:shrug: No preference either way.

Reply:
Alright, let's promote it now then. I'll think about how to best structure other extras (specifically the QIR dependency in dwave-gate) later.